src/features/amUI/users/UsersList.tsx

```diff
@@ -24,12 +32,43 @@ export const UsersList = () => {
     if (!rightHolders) {
       return [[], 1];
     }
-    const numPages = getTotalNumOfPages(rightHolders, pageSize);
-    return [getArrayPage(rightHolders, currentPage, pageSize), numPages];
-  }, [rightHolders, currentPage]);
+
```

Comment:
Nice implementation of the search! Its clear and concise, and it works great! 
We can consider optimizing with something like inverted indexes if we need to later if performance becomes an issue.